zds/tutorialv2/models/models_database.py

```diff
@@ -531,6 +548,16 @@ def add_tags(self, tag_collection):
 
         self.save()
 
+    def required_validation_before(self):
+        """
+        Check if content required a validation before publication.
+        Used to check if JsFiddle is available too.
+
+        :return: True if required a validation before publication, False else.
```

Comment:
"False otherwise"

Ou mieux : "Whether validation is required before publication"